pm/sendermonitor.go

```diff
@@ -125,15 +112,16 @@ func (sm *senderMonitor) AddFloat(addr ethcommon.Address, amount *big.Int) error
 	// Subtracting from pendingAmount = adding to max float
 	pendingAmount := sm.senders[addr].pendingAmount
 	if pendingAmount.Cmp(amount) < 0 {
-		return errors.New("cannot subtract from insufficient pendingAmount")
+		glog.Error(errors.New("cannot subtract from insufficient pendingAmount"))
```

Comment:
If we're logging this error then we don't need to return an error type in this method.

Also, we don't need to wrap the string in an error if we're using `glog.Error`